LEGO1/lego/legoomni/include/raceskel.h

```diff
@@ -3,19 +3,30 @@
 
 #include "legoanimactor.h"
 
-/*
-	VTABLE: LEGO1 0x100d7668 LegoPathActor
-	VTABLE: LEGO1 0x100d7738 LegoAnimActor
-*/
+// VTABLE: LEGO1 0x100d93f8 LegoPathActor
+// VTABLE: LEGO1 0x100d94c8 LegoAnimActor
+// VTABLE: BETA10 0x101bf9d0 LegoPathActor
+// VTABLE: BETA10 0x101bfac0 LegoAnimActor
 // SIZE 0x178
 class RaceSkel : public LegoAnimActor {
 public:
 	RaceSkel();
+	~RaceSkel() override;
+
+	void ParseAction(char* p_extra) override;
+
+	MxResult FUN_1001c360(float p_und, Matrix4& p_transform) override;
```

Comment:
Virtual functions (overrides or not) should have the // vtable+0x annotation

Reply:
Do we have a rule regarding multiple and virtual inheritance? It looks like the existing code is not entirely consistent about that either. For example, `LegoAnimActor` has no vtable annotations on the virtual functions it introduces. Another issue: `RaceSkel::FUN_1001c360` would have a `// vtable+0x04` but still come below `ParseAction` because it is in a different vtable.

I don't really mind how we want to do it, but the current pattern and rules aren't entirely clear to me.

Reply:
Ah yes, I forgot about the multiple inheritance. You are right that it's not consistent. We don't have any established rules for theses cases yet - considering this is limited to a few classes we may not need to have any. Let's just leave it as-is for now